extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ColumnWriterImpl.java

```diff
@@ -84,13 +86,15 @@ public class ColumnWriterImpl implements ColumnWriter {
                         getWidthFromMaxInt(column.getMaxRepetitionLevel()), MIN_SLAB_SIZE, targetPageSize, allocator);
         this.owner = owner;
         offsetIndexBuilder = OffsetIndexBuilder.getBuilder();
+        statistics = Statistics.createStats(column.getPrimitiveType());
```

Comment:
Make sure to test it out with all the types, and compare the statistics with pyarrow. There are examples in ParquetTableReadWriteTest.java and test_parquet.py which go over different data types.